src/brands/peapods/lib/PeaAccountProfile.js

```diff
@@ -33,6 +33,26 @@ const useStyles = makeStyles(() => ({
   followButton: {
     width: 160,
   },
+  scrollHeader: {
+    display: 'flex',
+    alignItems: 'center',
+    height: 50,
+    boxShadow: '3px 1px 20px rgba(0,0,0,0.2)',
+    padding: '0 10px',
+    position: 'sticky',
```

Comment:
Can you use position absolute instead? I don't want the component's height to change when the header becomes visible. Can you also animate it so that it slides in from the top?